File: Feitos em Python 3/2540.py
while True:
    try:
        times=int(input())
        vet =input().split()
        sizeVet=len(vet)
        impeachment=0
        i=0
        while i<sizeVet:
            if vet[i]=="1":
                impeachment+=1
            i+=1
        n=0
        n = (sizeVet*2.0) / 3.0
        if impeachment>=n:
            print("impeachment")
        else:
            print("acusacao arquivada")
    except EOFError:
        break

File: Feitos em C Sharp/2540.cs
using System;
namespace ConsoleApp16 {
    class Program {
        static void Main(string[] args) {
            while (true) {
                string times = Console.ReadLine();
                if (string.IsNullOrEmpty(times)) {
                    break;
                }
                string[] imput = Console.ReadLine().Split();
                int sizeInput = imput.Length;
                int[] vet = new int[sizeInput];
                int impeachment = 0;
                for(int i = 0; i < sizeInput; i++) {
                    vet[i] = int.Parse(imput[i]);
                    if (vet[i] == 1) {
                        impeachment++;
                    }
                }
                double n = (double) (sizeInput * 2) / 3;
                if (impeachment >=  n) {
                    Console.WriteLine("impeachment");
                }
                else {
                    Console.WriteLine("acusacao arquivada");
                }
            }
        }
    }
}